components/BadgeWithCopy.tsx

```diff
@@ -1,21 +1,19 @@
 import copy from "copy-to-clipboard";
 import { Copy } from "lucide-react";
+import { toast } from "sonner";
 import { Badge } from "./ui/badge";
-import { useToast } from "./ui/use-toast";
```

Comment:
Shouldn't we remove this function and file if we use a different lib fot toasts now?

Reply:
This lib was installed by shadcn so I was planning to remove it later with the rest of the unused components, using the shadcn CLI, after facelifting the rest of the app and getting rid of the old layout components too.

But yes, I could do it now.

Reply:
Okay, thanks. I don't mind much. Was just thinking we should not have two toast implementations in the app.